SlackAPI/SlackClientBase.cs

```diff
@@ -14,6 +14,14 @@ public abstract class SlackClientBase
         private readonly HttpClient httpClient;
         public string APIBaseLocation { get; set; } = "https://slack.com/api/";
 
+        static SlackClientBase()
+        {
+#if !NETSTANDARD1_3 && !NETSTANDARD1_6
```

Comment:
It seems there is no TLS 1.2 support for NetStandard 1.3 and 1.6 so we should probably drop NetStandard < 2 support

Reply:
Dropping support is fine for me